docs/text_handling.md

```diff
@@ -4,6 +4,19 @@ OpenVINO Model Server can now greatly simplify writing the applications with Nat
 
 We addressed both the situation when the original model requires tokens on input or output and there is added support for models with embedded tokenization layer. Below are demonstrated use cases with a simple client application sending and receiving text in a string format. Whole complexity of the text conversion is fully delegated to the remote serving endpoint.
 
+## Accepting strings by serving Python script
+
+When the model server is configured to serve python script (via MediaPipe Graph with PythonCalculator), it is possible to communicate with arbitrary format conforming Python's buffer protocol including string or list of strings. Refer to full [Python integration documentation](python_support/quickstart.md) or end-to-end [LLM text generation demo](../demos/python_demos/llm_text_generation/README.md).
+
+## Custom CPU extension for tokenization layer in the model
+
+Some AI model training frameworks supports the layers accepting the string format on the input or output. They include the layers performing the tokenization operations inside the neural network.
+While core OpenVINO does not contain the layers accepting string data type, it is possible to extend it with a CPU extension.
```

Comment:
It's a wider topic, but should we still refer to that as CPU extension?

Reply:
Unfortunately it still is a cpu extension

Reply:
I mean I can only see "CPU extension" in model server docs. In OpenVINO it's referred to as "custom operations". I wonder if this "CPU extension" name is historical leftover.

Reply:
I agree, and unfortunately it is --cpu_extension in OVMS parameters as well. I think we should stick to this name due to the param name